Blog-Articles/BrotatoBoi.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+
+<html>
+    <head>
+        <title>BrotatoBoi's Page!</title>
+
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+        <style>
+            body {
+                background-color: black;
+                color: lime;
+                text-align: center;
+            }
+
+            a {
+                color: deepskyblue;
+            }
+
+            a:visited {
+                color: red;
+            }
+
+            h1 {
+                margin-top: 5%;
+            }
+
+            img {
+                margin-top: 10%;
+                width: 100%;
+                height: 100px;
+            }
+
+            #main {
+                margin-top: 10%;
+                margin-left: 5%;
+                width: 90%;
+                height: fit-content;
+                border: 3px solid lime;
+            }
+        </style>
+    </head>
+    <body>
+        <h1>Welcome to my Page!</h1>
+
+        <div id="main">
+            <p>
+                I may be a little late to the party, but Happy Hacktober Everyone!
+                Hopefully next year I could participate for the whole month instead of the last few days...
+                I don't really expect anyone to see this but I might as well pretend it's visible!
+                If you want to check out my website you can visit <a href="https://brotatoboi.github.io/">here</a> :)
+            </p>
+        </div>
+
+        <img src="https://hacktoberfest.digitalocean.com/_nuxt/img/logo-hacktoberfest-full.f42e3b1.svg">
+    </body>
+</html>
```

Comment:
```html
<html>
<head>
<style>
body{
  background-color: white;
}

h1{
  color: #6F69AC;
  text-align: center;
  line-height: 0.3;
}

h5{
  color: #FD6F96 ;
  text-align: center;
}
p    {
  color: #716bad;
  font-family: Arial    ;
}
ol{
  margin-left: 20px;
}
li{
  color: #716bad;
  font-family: Monospace;
}
</style>
</head>
<body>
  <h1>Sum of Two Numbers</h1>
  <h5>By: John Doe</h5>
  <p>
    This algorithm uses bases addition operator to add two numbers
  </p>
  
  <ol>
    <li>Start</li>
    <li>Declare variables num1, num2 and sum</li>
    <li>Read values for num1, num2.</li>
    <li>Add num1 and num2 and assign the result to a variable sum.</li>
    <li>Display sum</li>
    <li>Stop</li>
  </ol>
</body>
</html>
```

We expect you to contribute algorithm of programs to our repository. You can create one on your own and support us... !!!